### ISSUE TITLE
Squashed focus indicator on InlineNotification

### ISSUE THREAD

**Issue by GodAImighty** [carbon-design-system/carbon#3973]
## Environment

> Operating system
Mac
> Browser
Chrome

## Detailed description

> What version of the Carbon Design System are you using?
v9
> What did you expect to happen?
Not squashed focus indicator
> What happened instead?
Squashed focus indicator
> What WCAG 2.1 checkpoint does the issue violate?
Perhaps 2.4.7, it kind of reduces visibility

## Steps to reproduce the issue

1. Create an inline notification with a really long subtitle
2. See this
![image](https://user-images.githubusercontent.com/22669933/64819278-7c9b0400-d573-11e9-9947-16c5c8a5a026.png)
  - Focus indicator on close button is squashed
